specs/simple-serialize.md

```diff
@@ -54,7 +55,7 @@ For convenience we alias:
 
 We recursively define the `serialize` function which consumes an object `value` (of the type specified) and returns a bytestring of type `"bytes"`.
 
-*Note*: In the function definitions below (`serialize`, `hash_tree_root`, `signed_root`, etc.) objects implicitly carry their type.
+> *Note*: In the function definitions below (`serialize`, `hash_tree_root`, `signed_root`, `is_fixed_size`, `is_variable_size`, etc.) objects implicitly carry their type.
```

Comment:
Maybe we can define `is_fixed_size` and `is_variable_size` after the previous section: 

> We recursively define "variable-size" types to be lists and all types that contains a variable-size type. All other types are said to be "fixed-size".

Reply:
It seems `is_fixed_size(element) := not is_variable_size(element)` or `is_variable_size(element) := not is_fixed_size(element)`? It could be simplified. If so, I suggest keeping `is_fixed_size` and removing `is_variable_size`.

Reply:
`signed_root` is renamed to `signing_root`.

Reply:
All comments in review addressed 👍 I kept `is_variable_size` because it's easier to define than `is_fixed_size`.